src/subdomains.ts

```diff
@@ -97,18 +70,36 @@ export default class implements Subdomains {
    * @param label - Subdomain to register. ie: alice
    * @param owner - The owner of the new subdomain
    */
-  async createSubdomain(domain: string, label: string, owner: string): Promise<void> {
-    this._validateDomain(domain);
-    this._validateLabel(label);
+  async setOwner(domain: string, label: string, owner: string): Promise<void> {
```

Comment:
I will rise a new issue: `register` should check available and then set owner, as a unique operation